packages/calling-component-bindings/package.json

```diff
@@ -29,9 +29,9 @@
   },
   "dependencies": {
     "@azure/communication-common": "2.0.0",
-    "@internal/acs-ui-common": "1.2.1-beta.0",
-    "@internal/calling-stateful-client": "1.2.1-beta.0",
-    "@internal/react-components": "1.2.1-beta.0",
+    "@internal/acs-ui-common": "1.2.2-beta.1",
+   "@internal/calling-stateful-client": "1.2.2-beta.1",
```

Comment:
nitty nit: indentation is out by one on this

Reply:
oops missed that! Just fixed it